src/java2yaml/Steps/RunJavadoc.cs

```diff
@@ -10,19 +13,145 @@ public RunJavadoc(string path)
             RepositoryPath = path;
         }
 
-        public string StepName => "RunJavadoc"; 
+        public string StepName => "RunJavadoc";
 
         public string RepositoryPath { get; }
 
+        private static ConfigModel _config;
+
         public Task RunAsync(ConfigModel config)
         {
             return Task.Run(() =>
             {
-                //TODO
+                Guard.ArgumentNotNullOrEmpty(RepositoryPath, nameof(RepositoryPath));
+
+                _config = config;
+
+                GenerateFileList(RepositoryPath);
+
+                var options = GenerateOptions(RepositoryPath);
+                var commandLineArgs = string.Concat(options, " @files");

```

Comment:
Better to reference constants: `{options} @{Constants.Files}`

Reply:
changed to $"{options} @{Constants.Files}"